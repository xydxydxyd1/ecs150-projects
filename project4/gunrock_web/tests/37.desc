Run ds3ls on all of the contents of a simple image
