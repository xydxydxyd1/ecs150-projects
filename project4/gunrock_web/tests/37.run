./ds3ls tests/disk_images/c.img /
