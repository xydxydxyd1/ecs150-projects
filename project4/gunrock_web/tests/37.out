0	.
0	..
1	file001.txt
2	file002.txt
3	file003.txt
4	file004.txt
5	file005.txt
6	file006.txt
7	file007.txt
8	file008.txt
9	file009.txt
10	file010.txt
11	file011.txt
12	file012.txt
13	file013.txt
14	file014.txt
15	file015.txt
16	file016.txt
17	file017.txt
18	file018.txt
19	file019.txt
20	file020.txt
21	file021.txt
22	file022.txt
23	file023.txt
24	file024.txt
25	file025.txt
26	file026.txt
27	file027.txt
28	file028.txt
29	file029.txt
30	file030.txt
31	file031.txt
32	file032.txt
33	file033.txt
34	file034.txt
35	file035.txt
36	file036.txt
37	file037.txt
38	file038.txt
39	file039.txt
40	file040.txt
41	file041.txt
42	file042.txt
43	file043.txt
44	file044.txt
45	file045.txt
46	file046.txt
47	file047.txt
48	file048.txt
49	file049.txt
50	file050.txt
51	file051.txt
52	file052.txt
53	file053.txt
54	file054.txt
55	file055.txt
56	file056.txt
57	file057.txt
58	file058.txt
59	file059.txt
60	file060.txt
61	file061.txt
62	file062.txt
63	file063.txt
64	file064.txt
65	file065.txt
66	file066.txt
67	file067.txt
68	file068.txt
69	file069.txt
70	file070.txt
71	file071.txt
72	file072.txt
73	file073.txt
74	file074.txt
75	file075.txt
76	file076.txt
77	file077.txt
78	file078.txt
79	file079.txt
80	file080.txt
81	file081.txt
82	file082.txt
83	file083.txt
84	file084.txt
85	file085.txt
86	file086.txt
87	file087.txt
88	file088.txt
89	file089.txt
90	file090.txt
91	file091.txt
92	file092.txt
93	file093.txt
94	file094.txt
95	file095.txt
96	file096.txt
97	file097.txt
98	file098.txt
99	file099.txt
100	file100.txt
101	file101.txt
102	file102.txt
103	file103.txt
104	file104.txt
105	file105.txt
106	file106.txt
107	file107.txt
108	file108.txt
109	file109.txt
110	file110.txt
111	file111.txt
112	file112.txt
113	file113.txt
114	file114.txt
115	file115.txt
116	file116.txt
117	file117.txt
118	file118.txt
119	file119.txt
120	file120.txt
121	file121.txt
122	file122.txt
123	file123.txt
124	file124.txt
125	file125.txt
126	file126.txt
127	file127.txt
128	file128.txt
129	file129.txt
130	file130.txt
131	file131.txt
132	file132.txt
133	file133.txt
134	file134.txt
135	file135.txt
136	file136.txt
137	file137.txt
138	file138.txt
139	file139.txt
140	file140.txt
141	file141.txt
142	file142.txt
143	file143.txt
144	file144.txt
145	file145.txt
146	file146.txt
147	file147.txt
148	file148.txt
149	file149.txt
150	file150.txt
151	file151.txt
152	file152.txt
153	file153.txt
154	file154.txt
155	file155.txt
156	file156.txt
157	file157.txt
158	file158.txt
159	file159.txt
160	file160.txt
161	file161.txt
162	file162.txt
163	file163.txt
164	file164.txt
165	file165.txt
166	file166.txt
167	file167.txt
168	file168.txt
169	file169.txt
170	file170.txt
171	file171.txt
172	file172.txt
173	file173.txt
174	file174.txt
175	file175.txt
176	file176.txt
177	file177.txt
178	file178.txt
179	file179.txt
180	file180.txt
181	file181.txt
182	file182.txt
183	file183.txt
184	file184.txt
185	file185.txt
186	file186.txt
187	file187.txt
188	file188.txt
189	file189.txt
190	file190.txt
191	file191.txt
192	file192.txt
193	file193.txt
194	file194.txt
195	file195.txt
196	file196.txt
197	file197.txt
198	file198.txt
199	file199.txt
200	file200.txt
